NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -155,7 +155,7 @@ public static IServiceCollection AddApplicationInsightsTelemetry(this IServiceCo
 
                     // NetStandard2.0 has a package reference to Microsoft.Extensions.Logging.ApplicationInsights, and
                     // enables ApplicationInsightsLoggerProvider by default.
-#if NETSTANDARD2_0
+#if NETSTANDARD2_0 || NET461
```

Comment:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/NETCORE/src/Shared/Extensions/ApplicationInsightsExtensions.cs#L449 This needs change as well for net461

Reply:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/develop/NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Extensions/RequestCollectionOptions.cs#L20  This too.

Reply:
I'd expect some functional tests to fail without above changes. There must be some test which validate if Exceptions are captured. In net46, exceptions were previously tracked by RequestTrackingModule,. Now, it should be disabled, as loggingprovider captures Exception.

Reply:
thanks for validating. The first one I added before. I will push the second one and rechecking the other as well